.eslintignore

```diff
@@ -1 +1,5 @@
 addon/StudyUtils.jsm
+dist
+OLD
```

Comment:
Instead of only linting /addon/ and /test/, we lint everything and use this as a blacklist.